### PR TITLE
Fixed issue #LB-60: Fixed JS part for copy functionality

### PR DESCRIPTION
**Pull request by apacheborys**
Fixed issue #18564: fixed handling response from BE

### CODE REVIEW

File: application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php [hunk 1]
```diff
@@ -29,12 +29,12 @@
                 code
               },
               success: (data) => {
-                if (data) {
-                    $('#question-title-warning').text(data);
-                    $('#question-title-warning').removeClass('hidden');
-                } else {
-                    $("#submit-copy-question").click();
-                }
+                  if (data.hasOwnProperty('message') && data.message === null) {
```
Comment (gabrieljenik) [inline application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php:32]: Didn't dig a lot into the issue, but just smell it... Shold the condition be like this maybe?
`!data.hasOwnProperty('message') || data.message === null`

Comment (apacheborys) [reply application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php:32]: According to changes on BE side, message item is mandatory. That's why I checking for item existence with null value. For me it seems like anti-pattern too. But we already have these changes in BE. WDYT?

application/controllers/QuestionAdministrationController.php:3036

image

Comment (gabrieljenik) [reply application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php:32]: So if there is no message property, data will be just showed as before. Also, this is already on develop? Seems OK then :)

**Review by gabrieljenik — APPROVED** [review]
Not the best pattern, but if it already working on develop, good!
Haven't tested it

File: application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php (hunk 1 above)
Comment (apacheborys) [reply application/extensions/TopbarWidget/views/includes/copyQuestionTopbarRight_view.php:32]: I'm not sure I got you correctly. Message property will be still permanently in response. And acceptance criteria to have a null as value (according to BE code). And yeah, it's already fixed in dev branch ✅

**Pull request merged**